cli/src/cli_builder.rs

```diff
@@ -615,6 +615,24 @@ fn xxx<'a>(command: CliCommand) -> Option<&'a str> {
         CliCommand::FloatingIpUpdate => Some("floating-ip update"),
         CliCommand::FloatingIpView => Some("floating-ip view"),
 
+        // Alert subcommands
+        CliCommand::WebhookEventClassList => Some("alert class list"),
+        CliCommand::WebhookReceiverList => Some("alert receiver list"),
+        CliCommand::WebhookDeliveryList => Some("alert receiver log"),
+        CliCommand::WebhookReceiverProbe => Some("alert receiver probe"),
+        CliCommand::WebhookDeliveryResend => Some("alert receiver resend"),
+        CliCommand::WebhookReceiverView => Some("alert receiver view"),
+        CliCommand::WebhookReceiverDelete => Some("alert receiver delete"),
+        CliCommand::WebhookReceiverSubscriptionAdd => Some("alert receiver subscribe"),
+        CliCommand::WebhookReceiverSubscriptionRemove => Some("alert receiver unsubscribe"),
+
+        // Webhook specific subcommands (including secret management)
+        CliCommand::WebhookReceiverCreate => Some("alert receiver webhook create"),
+        CliCommand::WebhookReceiverUpdate => Some("alert receiver webhook update"),
+        CliCommand::WebhookSecretsList => Some("alert receiver webhook secret list"),
+        CliCommand::WebhookSecretsAdd => Some("alert receiver webhook secret add"),
+        CliCommand::WebhookSecretsDelete => Some("alert receiver webhook secret delete"),
```

Comment:
@hawkw and @david-crespo would appreciate your review here

Reply:
I still feel kind of funny about the relationship between receivers and the webhook kind of receiver. I guess I get why the webhook subcommands are separate -- webhook receiver create and update are specific to webhooks, while view and delete aren't (at least in their inputs -- the output of view would have to be a union of the different types of receiver if there were more). It still doesn't feel quite natural or obvious, but maybe I just need to think about it more.

Reply:
I agree that it's a little speculative. My suggestion is that we pick something and hope it works out, but intend to reevaluate once we have a second kind of receiver.